src/OpenTelemetry.Exporter.Geneva/CHANGELOG.md

```diff
@@ -11,6 +11,11 @@
   [RS0026](https://github.com/dotnet/roslyn-analyzers/blob/main/src/PublicApiAnalyzers/Microsoft.CodeAnalysis.PublicApiAnalyzers.md#rs0026-do-not-add-multiple-public-overloads-with-optional-parameters).
   ([#1218](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/1218))
 
+* Fix the issue of running into the `ArgumentException`: `An instance of
```

Comment:
do we have same problem in log/trace exporter?

Reply:
Similar but slightly different (e.g. most folks can just to two different log ETW providers, while metrics always use the same provider).

Reply:
@reyang @cijothomas I was wondering if we need to worry about scenarios where there are multiple instances of an application running each with the same GenevaExporter setup. They would run into this issue.

Reply:
Actually, on second thoughts, maybe not? It's probably just tracked at a process level?